### PR TITLE
docs: rewriting ### Gnosis Auction User flow

### PR DESCRIPTION
**Pull request by josojo**
I like it much better if it gives a very high-level overview. Personally, in the old version, I would have stopped reading already once the auctioneer parameters are mentioned. They simply do not fit into the user flow for most users

### CODE REVIEW

**Comment by avsavsavs** [comment]
Everything looking great!!

File: src/docs/files/devguide04.md [hunk 1]
```diff
@@ -1,35 +1,34 @@
...
+#### 3. Auctioneer: On-chain price calculation
 
-_Important: as a bidder, it is important to note that many participants might submit new bids right before the auction closes, making it difficult to predict the closing price of the auction ahead of time. This, it is important for bidders to submit their bids with the highest price they are comfortable to pay for the asset_
+It's the auctioneers duty to finish the auction by sending the transaction that calculates the price of the auction on-chain. Though it's also a permissionless process and everyone can do it.
+
+Check out the [guide](https://github.com/gnosis/ido-contracts#settle-auctionss) for doing the price calculation.
+
+#### 4. Bidders: Claiming
+
+After the price has settled, the auctioneer will receive the proceeds of the auction. In contracts the bidder will need to submit an additional transaction to claim their auction proceeds.
```
Comment (josojo) [inline src/docs/files/devguide04.md:34]: itt should be
``` 
In contrast, ..

```

Comment (cmagan) [reply src/docs/files/devguide04.md:34]: Is there a way to make the images clickable so people can enlarge them? For the images are unusable, they are way too small to read

Comment (cmagan) [reply src/docs/files/devguide04.md:34]: Aside from that everything looks good. 
@josojo I liked the whitelist section

**Review by cmagan — COMMENTED** [review]
Changed a bit the text in the Overview section
Everything looks good

File: src/docs/files/devguide04.md (hunk 1 above)
Comment (josojo) [reply src/docs/files/devguide04.md]: Yes, I agree that htis is a necessary feature. I will create an issue

**Pull request merged**